lib/drone/chef/config.rb

```diff
@@ -18,8 +19,9 @@ class Config
       #
       # Initialize an instance
       #
-      def initialize(payload)
+      def initialize(payload, log = nil)
         self.payload = payload
+        @logger = log || logger
```

Comment:
I would define an attr_accessor like it's done for payload and set it with self.logger, than it looks pretty much the same.